bin/check_fixmystreet

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env perl
+
+use strict;
+use lib '/usr/lib/nagios/plugins';
+use utils qw(%ERRORS);
+use Getopt::Long;
+use FixMyStreet::DB;
```

Comment:
This and next two lines will need to be under the BEGIN block below because they can't be found before it (sorry if I wasn't clear enough when I said "just above where you set $reports and $updates").

Reply:
You were clear, and I should have realised that anyway.